commons/com.b2international.index.tests/src/com/b2international/index/StreamTest.java

```diff
@@ -104,9 +104,11 @@ public void searchAndReturnAllHitsWithStream() throws Exception {
 		List<Data> hits = newArrayListWithExpectedSize(NUM_DOCS);
 		
 		Stopwatch w = Stopwatch.createStarted();
+		
+		final int pageSize = Integer.parseInt((String) getIndexSettings().getOrDefault(IndexClientFactory.RESULT_WINDOW_KEY, "" + IndexClientFactory.DEFAULT_RESULT_WINDOW));
 		final Query<Data> query = Query.select(Data.class)
 			.where(Expressions.matchAll())
-			.limit(1000)
+			.limit(pageSize)
```

Comment:
I think the purpose here was to actually page through the results with the page size of `1000`, now we are going to fetch it in one go, right?